SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -27,6 +28,8 @@ enum class ESpatialConnectionType
 	Locator
 };
 
+typedef std::function<bool(const Worker_Alpha_LoginTokensResponse*)> LoginTokenCb;
```

Comment:
Would it be possible to use  TFunction here? It is the more unreal way to do this kind of thing.